src/Components/ArticlesPageComponents/ArticlesList.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+import { useEffect } from "react";
+import { getArticles } from "../../api";
+import axios from "axios";
+import ArticleCard from "./ArticleCard";
+
+const ArticlesList =()=>{
+
+const [articlesList, setArticlesList] = useState([])
+const [isLoading, setIsLoading] = useState(true)
+
+const articlesApi = axios.create({
+    baseURL: "https://v2-nc-news-backend-project-northcoders.onrender.com"
+})
+useEffect(()=>{
+   articlesApi.get("/api/articles").then(({data})=>{
+        console.log(data.articles)
+        setIsLoading(false)
+        setArticlesList(data.articles)
+    })
+}, [])
+    return (<section >
+        <h2>Articles</h2>
+        <section id="articlesList">
+            {articlesList.map((article)=>{
+                return ( 
+                <li key={article.articleId}>
```

Comment:
`<li>` should be wrapped in an `<ul>` or an` <ol>` tag. Also it would be better if the `<li> `tags are in `<ArticleCard>`
There's a warning about unique key, putting unique key in` <ArticleCard key={article.articleId} article={article} /> `would fix the issue.

Reply:
have made these changes but the warning is still there unfortunately!